ddtrace/llmobs/_llmobs.py

```diff
@@ -167,10 +168,10 @@ def enable(
                     "DD_SITE is required for sending LLMObs data when agentless mode is enabled. "
                     "Ensure this configuration is set before running your application."
                 )
-            if not os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED"):
-                config._telemetry_enabled = False
-                log.debug("Telemetry disabled because DD_LLMOBS_AGENTLESS_ENABLED is set to true.")
-                telemetry.telemetry_writer.disable()
+            if telemetry.telemetry_writer.enable():
+                telemetry.telemetry_writer.enable_agentless_client()
+                telemetry.telemetry_writer.product_activated (TELEMETRY_APM_PRODUCT.LLMOBS, True)
```

Comment:
We don't need to check if telemetry is enabled. As long as `DD_INSTRUMENTATION_TELEMETRY_ENABLED` is NOT set to `False` telemetry will be enabled when the first event is queued.
 
```suggestion
            telemetry.telemetry_writer.enable_agentless_client()
            telemetry.telemetry_writer.product_activated (TELEMETRY_APM_PRODUCT.LLMOBS, True)
```